sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/sweep.py

```diff
@@ -57,49 +57,32 @@
 
 
 class Sweep(ParameterizedSweep, BaseNode):
-    """Base class for sweep node, used for command component version
-    consumption.
+    """Base class for sweep node.
 
-    You should not instantiate this class directly. Instead, you should
-    create from builder function: sweep.
+    This class should not be instantiated directly. Instead, it should be created via the builder function: sweep.
 
-    :param trial: Id or instance of the command component/job to be run for the step
-    :type trial: Union[CommandComponent, str]
-    :param compute: Compute definition containing the compute information for the step
+    :param trial: The ID or instance of the command component or job to be run for the step.
+    :type trial: Union[~azure.ai.ml.entities.CommandComponent, str]
+    :param compute: The compute definition containing the compute information for the step.
     :type compute: str
-    :param limits: limits in running the sweep node.
-    :type limits: SweepJobLimits
-    :param sampling_algorithm: sampling algorithm to sample inside search space
-    :type sampling_algorithm: str, valid values: random, grid or bayesian
-    :param objective: the objective used to pick target run with the local optimal hyperparameter in search space.
-    :type objective: Objective
-    :param early_termination_policy: early termination policy of the sweep node.
-    :type early_termination_policy: Union[
-    ~azure.mgmt.machinelearningservices.models.BanditPolicy,
-    ~azure.mgmt.machinelearningservices.models.MedianStoppingPolicy,
-    ~azure.mgmt.machinelearningservices.models.TruncationSelectionPolicy]
-    :param search_space: hyperparameter search space to run trials.
-    :type search_space: Dict[str, Union[~azure.ai.ml.entities.Choice,
-    ~azure.ai.ml.entities.LogNormal,
-    ~azure.ai.ml.entities.LogUniform,
-    ~azure.ai.ml.entities.Normal,
-    ~azure.ai.ml.entities.QLogNormal,
-    ~azure.ai.ml.entities.QLogUniform,
-    ~azure.ai.ml.entities.QNormal,
-    ~azure.ai.ml.entities.QUniform,
-    ~azure.ai.ml.entities.Randint,
-    ~azure.ai.ml.entities.Uniform]]
-    :param inputs: Inputs to the command.
-    :type inputs: Dict[str, Union[Input, str, bool, int, float]]
+    :param limits: The limits for the sweep node.
+    :type limits: ~azure.ai.ml.sweep.SweepJobLimits
+    :param sampling_algorithm: The sampling algorithm to use to sample inside the search space. Accepted values are: "random", "grid", or "bayesian". # pylint: disable=line-too-long
```

Comment:
The pylint directive _might_ end up rendering in the reference docs. I'm pretty sure it's getting included in the docstring.

Sphinx docstrings are valid restructuredText documents, so I'm pretty sure you can break up the line like this:


```suggestion
    :param sampling_algorithm: The sampling algorithm to use to sample inside the search space. 
        Accepted values are: "random", "grid", or "bayesian".
```

Reply:
There are a couple more instances of `pylint: disable=line-too-long` in this PR that may need a follow up.